src/apiCalls.js

```diff
@@ -1,26 +1,18 @@
-const getAllArt = () => {
-  try {
-		return fetch("https://www.rijksmuseum.nl/api/en/collection?key=AgQXh8Og&involvedMaker=Rembrandt+van+Rijn")
-		.then(response => response.json())
-		.then(data => data)
-		
-	} catch (error) {
-		console.log(error);
+const checkResponse = (response) => {
+	if (!response.ok) {
+		throw new Error(response.statusText);
```

Comment:
hey, I see that you throw an error here how are you handling it? the application won't work properly if not handled.